src/libNode/Node.cpp

```diff
@@ -735,22 +735,24 @@ bool Node::ProcessTxnPacketFromLookupCore(
                 else
                 {
                     compIdx.insert(tx);
-                    txn_sent_count++;
                 }
+                txn_sent_count++;
```

Comment:
Put ++ in front will be more efficient.

Reply:
I'm afraid there's no difference in terms of the efficiency between `txn_sent_count++` and `++txn_sent_count`.
This is an example in `-O2` (also the same in `-O2`) https://godbolt.org/z/ci-DHi. Let's just trust the compiler can do better work on this.